package.json

```diff
@@ -274,7 +275,9 @@
     "mixedindentlint": "1.2.0",
     "nock": "8.0.0",
     "nodemon": "1.4.1",
+    "pad": "1.2.0",
```

Comment:
Could we not add yet another dependency for something as basic as `pad`. Lodash has a [`pad` function](https://lodash.com/docs/4.17.4#pad)

Reply:
Good point, I'll remove it!